test/wasmBenchmarker/ctests/fibonacci.c

```diff
@@ -19,17 +19,15 @@
 
 uint64_t fibonacci(uint64_t n) {
     if (n == 0 || n == 1) {
-        return 1;
+        return n;
     }
 
-    fibonacci(n - 1);
-    fibonacci(n - 2);
 
-    return fibonacci(n - 2);
+    return fibonacci(n-1) + fibonacci(n-2);
 }
 
 uint64_t runtime() {
-    return fibonacci(26);
+    return fibonacci(39);
```

Comment:
Does the result fit into 64 bit? Fibonacci generates pretty big numbers.

Reply:
Yes, 63245986 fits into 64 bits.